scripts/install/install_unix.sh

```diff
@@ -768,7 +753,7 @@ display_results()
       info "Status Command:     $(fg_cyan "sudo service observiq-otel-collector status")$(reset)"
     fi
     info "Logs Command:       $(fg_cyan "sudo tail -F /opt/observiq-otel-collector/log/collector.log")$(reset)"
-    info "Uninstall Command:  $(fg_cyan "sudo sh -c \"\$(curl -fsSlL https://github.com/observIQ/bindplane-otel-collector/releases/latest/download/install_unix.sh)\" install_unix.sh -r")$(reset)"
+    info "Uninstall Command:  $(fg_cyan "sudo sh -c \"\$(curl -fsSlL https://${DOWNLOAD_BASE}/${version}/install_unix.sh)\" install_unix.sh -r")$(reset)"
```

Comment:
Uninstall command has the correct URL and path. 

<img width="947" height="237" alt="Screenshot 2025-07-19 at 3 50 45 PM" src="https://github.com/user-attachments/assets/50857e57-6837-4807-8437-b7c45fca6bef" />